tests/test_simresults.py

```diff
@@ -244,7 +244,7 @@ def test_expect(results, pi_pulse, reg):
     config = SimConfig(noise="SPAM", eta=0)
     sim_single.set_config(config)
     sim_single.set_evaluation_times("Minimal")
-    results_single = sim_single.run()
+    results_single = sim_single.run(nsteps=2000)
```

Comment:
Is this because you were getting an ODE error now?

Reply:
I was getting a Warning followed by an error:
UserWarning: "zvode: Excess work done on this call (Perhaps wrong MF.)": 
Exception: ODE integration error: Try to increase the allowed number of substeps by increasing the nsteps parameter in the Options class.

At least the error is explicit

Reply:
I guess we can try to suggest a nsteps from the samples of the Sequence as well

Reply:
That was not an hard issue to solve ^^'